api/routes/analysis.py

```diff
@@ -45,7 +45,7 @@ class AnalysisModel(BaseModel):
     type: str
     status: AnalysisStatus
     meta: dict[str, Any] | None = None
-    output: str | None = None
+    outputs: str | None = None
```

Comment:
Should not outputs be of type Union[dict, str] ?

Reply:
You're right, somehow missed this since I've been interacting from the layer level directly! I've also just reviewed all other type annotations for the `Analysis` model to use `Union, Optional` instead of the `|` operator.